src/main/scala/beam/sim/BeamServices.scala

```diff
@@ -36,14 +38,14 @@ case class BeamServices @Inject()(protected val injector: Injector) extends Acto
   var physSim: ActorRef = _
   var schedulerRef: ActorRef =_
   var taxiManager: ActorRef = _
-  var persons: Map[Id[Person], Person] = Map()
-  var personRefs: Map[Id[Person], ActorRef] = Map()
-  var vehicles: Map[Id[Vehicle], Vehicle] = Map()
-  var vehicleRefs: Map[Id[Vehicle], ActorRef] = Map()
-  var households : Map[Id[Household], Household] = Map()
-  var householdRefs : Map[Id[Household], ActorRef] = Map()
-  var agentRefs: Map[String, ActorRef] = Map()
-  var transitVehiclesByBeamLeg: Map[BeamLeg, Id[Vehicle]] = Map()
+  val persons: mutable.Map[Id[Person], Person] = new ConcurrentHashMap[Id[Person], Person]().asScala
```

Comment:
I'm unfamiliar with ConcurrentHashMap but why create a Java object and immediately convert to mutable.Map? Why no just create a mutable.Map?